### PR TITLE
IGNITE-14281 Calcite. Colocated tables are considered non colocated

### PR DESCRIPTION
**Pull request by korlov42**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/core/src/test/config/log4j-test.xml [hunk 1]
```diff
@@ -141,6 +141,10 @@
         <level value="INFO"/>
     </category>
 
+    <category name="org.apache.calcite">
```
Comment (tledkov-gridgain) [inline modules/core/src/test/config/log4j-test.xml:144]: What is this category used for? The previous category does the same.

File: modules/calcite/src/test/java/org/apache/ignite/testsuites/IgniteCalciteTestSuite.java [hunk 2]
```diff
@@ -42,7 +42,7 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
     PlannerTestSuite.class,
-    ExecutionTestSuite.class,
+//    ExecutionTestSuite.class,
```
Comment (tledkov-gridgain) [inline modules/calcite/src/test/java/org/apache/ignite/testsuites/IgniteCalciteTestSuite.java:45]: Disabled for debug?

**Review by tledkov-gridgain — APPROVED** [review]

File: modules/core/src/test/config/log4j-test.xml (hunk 1 above)
Comment (korlov42) [reply modules/core/src/test/config/log4j-test.xml]: removed

File: modules/calcite/src/test/java/org/apache/ignite/testsuites/IgniteCalciteTestSuite.java (hunk 2 above)
Comment (korlov42) [reply modules/calcite/src/test/java/org/apache/ignite/testsuites/IgniteCalciteTestSuite.java]: reverted

**Pull request closed without merging**